docs/use_cases/model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_WeatherRegime.py

```diff
@@ -1,186 +1,193 @@
 """
-WeatherRegime Calculation: ERA RegridDataPlane, PcpCombine, and WeatherRegime python code
-==========================================================================================
+UserScript: Calculate Weather Regimes for ERA and possibly RegridDataPlane and PcpCombine
+=========================================================================================
 
 model_applications/
```

Comment:
```suggestion
model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_WeatherRegime.py
```
The [template](https://github.com/dtcenter/METplus/blob/develop/docs/use_cases/use_case_documentation_template.py) has the full path on one line.